taca/analysis/analysis_element.py

```diff
@@ -39,7 +39,7 @@ def _process(run):
         sequencing_done = run.check_sequencing_status()
         if not sequencing_done:
             run.status = "sequencing"
-            if run.status_changed:
+            if run.status_changed():
```

Comment:
I think(?) this is a bugfix

Reply:
Yes I think you're right. But my search found 5 other `run.status_changed` without the function call...

Reply:
[05979c7](https://github.com/ssjunnebo/TACA/pull/21/commits/05979c7208b377d60d569a3a79512639ea84e53f)

Reply:
Thank you!